x/farm/client/cli/tx.go

```diff
@@ -84,19 +85,25 @@ $ %s tx %s create-private-plan "New Farming Plan" 2022-01-01T00:00:00Z 2023-01-0
 			var rewardAllocs []types.RewardAllocation
 			for _, arg := range args[3:] {
 				// TODO: use strings.Cut with go 1.18
```

Comment:
As we just upgraded go version to 1.18, I think we can use `strings.Cut` if you pull the latest `main` branch. I am leaving this comment for a reminder. You can tackle this in a separate PR.